src/context/alert.tsx

```diff
@@ -19,7 +19,6 @@ export enum ERROR_MESSAGE {
   DUPLICATE_POD_INPUT = "Scan another item that is not tagged to any ID number.",
   INVALID_IDENTIFIER_INPUT = "Enter or scan valid code details.",
   MISSING_IDENTIFIER_INPUT = "Enter or scan code details.",
-  INVALID_VOUCHER_INPUT = "Enter valid voucher code details.",
```

Comment:
Was this intended?

Reply:
it wasn't being used in any component and we were using this in the CustomerQuotaScreen:
```case ERROR_MESSAGE.INVALID_IDENTIFIER_INPUT:
          showAlert({
            ...wrongFormatAlertProps,
            description:
              campaignFeatures?.campaignName === "TT Tokens"
                ? ERROR_MESSAGE.INVALID_POD_INPUT
                : ERROR_MESSAGE.INVALID_IDENTIFIER_INPUT,
            onOk: () => clearError()
          });
          break;```

Reply:
Eh so is this for campaign name is vouchers? I am confused that you said it isn't being used but it is being used in CustomerQuotaScreen

Reply:
I think the alert will show either the invalid pod input for the TT token dist and for invalid input for other dist (voucher,FFTH,EG) it will show the general INVALID_IDENTIFIER_INPUT enum. But i can add the enum back in if you want also!

Reply:
No if it's not being used let's remove it! But wanted to make sure we are not removing something that was originally used only

Reply:
Maybe can confirm with @dalsontws and @ddwsd too

Reply:
Can resolve whenever you confirm with them~